### ISSUE TITLE
Win/lose notifcation

### ISSUE THREAD

**Issue by garycarlyle** [rolyatmax/tictactoe#9]
Would be useful to be told when you have one or lost and for it to pause for a moment on the final board. 
We never get to see the final board and sometimes the result is not obvious partly because of that.

**Comment by rolyatmax** [comment]
Yeah - I totally agree! I'm afraid I'm not really making any updates to this project at the time, but if you'd like to submit a pull request, I'm happy to merge!
